modules/programs/firefox.nix

```diff
@@ -663,6 +680,19 @@ in {
           (", namely " + concatStringsSep ", " defaults);
       })
 
+      (let
+        getContainers = profiles:
+          flatten (mapAttrsToList (_: value: value.containers) profiles);
+
+        findInvalidContainerIds = profiles:
+          filter (container: container ? "id" && container.id < 4294967294)
+          (getContainers profiles);
+      in {
+        assertion = cfg.profiles == { }
+          || length (findInvalidContainerIds cfg.profiles) != 0;
```

Comment:
I think `!= 0` should be changed to `== 0`.

Reply:
You are right but also had to switch the filter function.